overlord/snapstate/snapstate.go

```diff
@@ -68,8 +69,12 @@ const (
 	HooksEdge                 = state.TaskSetEdge("hooks")
 )
 
+type ErrInsufficientSpace struct{ error }
```

Comment:
I think this should look a bit more like ChangeConflictError perhaps

Reply:
Done